api/v1/src/accounts/index.js

```diff
@@ -682,11 +743,241 @@ accounts.get(['/projects/:projectId/accounts:register', '/accounts:register'], a
     }
 });
 
-// Backwards compatibility for marketplace
-accounts.post(['/projects/:projectId/accounts::custom', '/accounts::custom'], async (req, res) => {
+/**
+ * @swagger
+ *
+ * /accounts:register:
+ *   post:
+ *     summary: Register a marketplace account based off request body
+ *     description: Returns a redirect response
+ *     operationId: registerAccount
+ *     security: [] # no security for marketplace register post request
+ *     tags:
+ *       - accounts
+ *     parameters:
+ *     - in: header
+ *       name: x-gcp-project-id
+ *       type: string
+ *       required: true
+ *     - in: body
+ *       name: account
+ *       description: Request parameters for Account
+ *       schema:
+ *         $ref: '#/definitions/Account'
+ *     responses:
+ *       201:
+ *         description: Account
+ *         schema:
+ *           type: object
+ *           properties:
+ *             success:
+ *               type: boolean
+ *               description: Success of the request
+ *             code:
+ *               type: integer
+ *               description: HTTP status code
+ *             data:
+ *               type: object
+ *               items:
+ *                 $ref: '#/definitions/Account'
+ *       404:
+ *         description: Error
+ *         schema:
+ *           $ref: '#/definitions/Error'
+ *       500:
+ *         description: Error
+ *         schema:
+ *           $ref: '#/definitions/Error'
+ */
+/**
+ * @swagger
+ *
+ * /accounts:activate:
+ *   options:
```

Comment:
I don't understand yet why this options is needed, though without it API gateway denies the post.